higher_health/templates/healthcheck_questionnaire.html

```diff
@@ -136,6 +136,10 @@ <h3 class="heading-subtitle heading-subtitle--edu-label">Confirmation</h3>
                   </li>
                 {% endfor %}
               </ul>
+              {% if form.errors %}
+                <br/>
+                <p class="error">Please correct all the errors before submitting.</p>
+              {% endif %}
```

Comment:
Shouldn't this be with the button, rather than inside the medical `div` with a `br`?

Reply:
With the button it appears outside the form, which doesn't look right. 
![image](https://user-images.githubusercontent.com/623417/83752403-c8915a00-a668-11ea-87db-8cae87f858f7.png)

Reply:
Ah right, we probably want it in the white part. 👍